motion_computation/test/motion_computation_external_object_launch_test.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2023 Leidos
```

Comment:
how do we run these tests?

Reply:
By hand. I modeled it off of autoware.univere's documentation. I'm not sure if they have an auto-generator for these types of file. I don't think they do since it varies from package to package.

I think this style of documentation should be standard in all our packages because it provides users a quick way to see a Node's interface.

Reply:
They get run automatically when you invoke `colcon test`. Here's the relevant snippet from the package's `CMakeLists.txt` file: 

```cmake
add_launch_test(test/motion_computation_external_object_launch_test.py)
add_launch_test(test/motion_computation_queueing_launch_test.py)
```

Reply:
okay thanks